language-design.md

```diff
@@ -62,7 +129,37 @@ the commenters?
 
 **Response**
 
-
+The essence of the conflict between professional programmers and everyone is
+that the professional programmers have seen many bad attempts at making coding
+"easier" and are fearful of being forced to use another. In the attempt to make
+programming easier, languages such as Scratch have been developed, where it's
+fairly easy to do atomic tasks that the developers thought of, but are, as
+Steele would put it, unable to grow. So many bad attempts have been made, ones
+that just cater to new users but are painfully hard to do anything advanced in,
+that the professional programmers have become wary of the attempts. In short,
+the programmers are afraid of oversimplification so much that they support no
+simplification at all.
+
+While the fear of oversimplification may not be entirely rational, there are
+dangers with the movement of programming languages closer to natural languages.
+Natural languages are almost inherently ambiguous and in class most people
+agreed that programming languages should be unambiguous. Thus, to use a natural
+language as a programming language, somewhat arbitrary decisions must be made
+to remove the ambiguity, which can make it harder to write because one must
+know about these decisions to program correctly.
+
+On the other hand, simplification of programming languages is a worthwhile task
+and is certainly _possible_ to do without oversimplifying them. Professional
+programmers should not immediately dismiss attempts at simplification since if
+attempting is discouraged, no progress will be made.
+
+What's interesting, though, is that programming languages are already not all
+that difficult to learn: as commenter Peter says, "In particular, you can learn
+C++, say, much more easily than you can a new natural language." Sure, there are
+many rabbit-holds that can lead you to years of studying a language, and some
+communities are not particularly welcoming to newcomers, but programming
+languages _were_ designed for people to be able to use and are still relatively
+good at that.
 
```

Comment:
Interesting. We didn't think of it as oversimplification so much as making code readable to the general public at the expense of making it harder to do things than before. I agree that the author is arguing for programming languages closer to natural languages. And we definitely agree about how there are programming languages that are good at people being able to use them.